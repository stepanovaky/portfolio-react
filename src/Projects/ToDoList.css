.to-do-list img {
    display: block;
    margin-left: auto;
    margin-right: auto;
    height: 200px;
    width: 200px;
    border: 2px solid rgb(75, 5, 75);
    border-radius: 5%;
}

.to-do-list p {
    margin: 1em;
}

