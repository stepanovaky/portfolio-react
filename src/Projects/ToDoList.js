import React from 'react';
import ToDoListScreenshot from '../Images/To-Do-List-Screenshot.png';
import './ToDoList.css';

function ToDoList() {
    return (
        <div className="to-do-list">
            <h2>To Do List</h2>
            <img src={ToDoListScreenshot} alt="screenshot of To Do List" />
            <p>Keep yourself organized with this To Do List</p>
            <p>This To Do List was created to help users organize their tasks based on level of importance.</p>
            <p>I was inspired to create this app based on the Eisenhower Model, which has helped me numerous times in the past.</p>
            <p>I used HTML, CSS, JavaScript and jQuery to create this app.</p>
            <p><a href="https://stepanovaky.github.io/to-do-list/">Live Link</a></p>
            <p><a href="https://github.com/stepanovaky/to-do-list">Link to Github Repository</a></p>
        </div>
    )
}

export default ToDoList;