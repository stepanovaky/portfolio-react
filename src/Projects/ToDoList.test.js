import React from 'react';
import renderer from 'react-test-renderer';
import ToDoList from './ToDoList';

it('renders correctly', () => {
    const tree = renderer
        .create(<ToDoList />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})