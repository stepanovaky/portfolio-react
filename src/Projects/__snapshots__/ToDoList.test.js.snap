// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="to-do-list"
>
  <h2>
    To Do List
  </h2>
</div>
`;
